src/util/generatePreview.js

```diff
@@ -38,6 +39,19 @@ const errorHandlerScript = `(${(() => {
   };
 }).toString()}());`;
 
+const alertReplacementScript = `(${(() => {
+  // Object.create so our reference
+  // doesn't get destroyed when we delete it on the window
+  const _swal = Object.create({swal: window.swal});
```

Comment:
Hmm, not sure I understand the need for the `Object.create` dance here—can’t we just declare a local variable and assign `window.swal` as its value?